src/scales/scale.linear.js

```diff
@@ -161,26 +161,17 @@ module.exports = LinearScaleBase.extend({
 		// This must be called after fit has been run so that
 		// this.left, this.top, this.right, and this.bottom have been defined
 		var me = this;
-		var start = me.start;
-
+		var start = me.min;
 		var rightValue = +me.getRightValue(value);
-		var pixel;
-		var range = me.end - start;
-
-		if (me.isHorizontal()) {
-			pixel = me.left + (me.width / range * (rightValue - start));
-		} else {
-			pixel = me.bottom - (me.height / range * (rightValue - start));
-		}
-		return pixel;
+		var range = me.max - start;
+		return me.getPixelForDecimal((rightValue - start) / range);
```

Comment:
`getPixelForValue` is called for every data point, which makes it rather performance-sensitive in large charts. I wonder if we really want to add a function call to call `getPixelForDecimal` when it's only one line of code:
```
me._start + (me._reverse ? 1 - decimal : decimal) * me._length
```

Reply:
its negligible: https://jsperf.com/function-vs-inlined
IMO maintainability weights more here (vs having same code in each scale)